src/plugins/auto/docs/tests.md

```diff
@@ -1,9 +1,52 @@
 # AUTO Plugin tests
 
-## AUTO plugin testing places
+## AUTO plugin Unit Tests
 
-TBD-100527
+Auto unit test is a set of unit tests using gmock, each of which is for testing selection logic, device list parser, meta device filter and all other internal functions of Auto plugin.
 
 ## How to run tests
 
-TBD-100527
\ No newline at end of file
+### Build unit test
+
+1. Turn on `ENABLE_TESTS` in cmake option:
+
+   ```bash
+   cmake -DCMAKE_BUILD_TYPE=Release \
+       -DENABLE_TESTS=ON \
+   ```
+
+2. Build
+
+   ```bash
+   make ieMultiPluginUnitTests
```

Comment:
should we rename executable to be `ov_multi_unit_tests` ?
Or even, since we deprecated MULTI in favor of cumulative tput mode, should we just have `ov_auto_unit_tests` ?